planner/plan.go

```diff
@@ -32,6 +32,7 @@ type Plan struct {
 
 type Stage struct {
 	Command string `cue:"string" json:"command"`
+	Image   string `json:"-"`
```

Comment:
I prefer removing this in favor of https://github.com/jetpack-io/devbox/pull/52

Reply:
I'll change it once #52 is merged (might be follow up)